fix(a11y): final fix for WCAG 1.1.1 image alt should describe image content accurately (#17)
scottrigby/basekamp-next: content/events/ithaca-we-trust.md

title: In Ithaca We Trust
images:
  - src: hourpile2.jpg
    alt: Collection of Ithaca Hours local currency bills in various colorful denominations (quarter hour, eighth hour, etc.) arranged overlapping to display designs and portraits
  - src: paultiedyebigcrop.png
    alt: Portrait photograph of Paul Glover, founder of Ithaca Hours local currency system
  - src: ithpowercover.jpg
    alt: Book cover titled Ithaca Power with illustration of people, wind turbine, tree, and building on tan background by Paul Glover
  - src: book.jpg
    alt: Book cover with yellow background showing Health Democracy text and sun logo, Liberating Americans from Medical Insurance by Paul Glover
  - src: labornote.jpg
    alt: Historical labor note showing Three Hours Labour in Carpenter's Work with ornate frame, Justice label, and text Sample Labor Note
  - src: gjp.jpg

>     alt: Book cover with green background reading Green Jobs Philly about 100,000 new jobs to rebuild Philadelphia neighborhoods and clean environment by Paul Glover

Reviewer: ```suggestion
    alt: Book cover reading Green Jobs Philly subtitle 100,000 new jobs to rebuild Philadelphia neighborhoods and clean environment by Paul Glover
```
